phygnn/layers/custom_layers.py

```diff
@@ -9,10 +10,9 @@
 
 
 class FlexiblePadding(tf.keras.layers.Layer):
-    """Class to perform padding on tensors
-    """
+    """Class to perform padding on tensors"""
 
-    def __init__(self, paddings, mode='REFLECT', option='tf'):
+    def __init__(self, paddings, mode="REFLECT", option="tf"):
```

Comment:
Why the change in quote style? I greatly prefer single quote because it's one less key press.

Reply:
This was just an auto format change. I'll change it back.